src/js/views/globals/search/patient-search_views.js

```diff
@@ -38,52 +42,176 @@ const EmptyView = View.extend({
     this.listenTo(this.state, 'change:search', this.render);
   },
   templateContext() {
-    const settings = this.state.get('settings');
-
     return {
-      identifiers: settings && settings.identifiers,
+      settings: this.state.get('settings'),
     };
   },
   getTemplate() {
     const search = this.state.get('search');
 
     if (!search || search.length < 3) return TipTemplate;
-    if (this.collection.isSearching) return hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.searching }}`;
+    if (this.collection.isSearching) return hbs`{{ @intl.globals.search.patientSearchViews.emptyView.searching }}`;
 
-    return hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.noResults }}`;
+    return hbs`{{ @intl.globals.search.patientSearchViews.emptyView.noResults }}`;
   },
 });
 
-const PatientSearchPicklist = Picklist.extend({
-  className: 'patient-search__picklist',
-  getItemSearchText() {
-    return `${ this.model.get('first_name') } ${ this.model.get('last_name') }`;
+const PicklistItem = View.extend({
+  tagName: 'li',
+  className: 'js-picklist-item patient-search__picklist-item',
+  initialize({ state }) {
+    this.state = state;
+    this.listenTo(this.state, 'change:search', this.render);
+  },
+  triggers: {
+    'click': 'select',
   },
-  itemClassName: 'patient-search__picklist-item',
-  itemTemplate: hbs`
-    {{matchText text search}}{{~ remove_whitespace ~}}
-    <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
+  onSelect() {
+    this.state.set({ selected: this.model });
+  },
+  template: hbs`
+    <div class="patient-search__picklist-item-name u-text--overflow">
+      <span>{{matchText name search}}{{~ remove_whitespace ~}}</span>
+    </div>
+    <div class="patient-search__picklist-item-meta">
+      <div class="patient-search__picklist-item-dob u-text--overflow">
+        {{formatDateTime birth_date "MM/DD/YYYY"}}
+      </div>
+      {{#if identifiers.0.value}}
+        <div class="patient-search__picklist-item-identifier u-text--overflow">
+          {{matchText identifiers.0.value search}}
+        </div>
+      {{/if}}
+    </div>
   `,
-  itemTemplateContext() {
+  templateContext() {
     return {
-      text: this.getItemSearchText(),
+      name: `${ this.model.get('first_name') } ${ this.model.get('last_name') }`,
       search: this.state.get('search'),
     };
   },
+});
+
+const ListView = CollectionView.extend({
+  className: 'flex-region',
+  behaviors: [
+    PicklistBehavior,
+  ],
```

Comment:
This behavior is already on the `DialogView` view.

Does it need to be set again?